openage/convert/interface/interfacerename.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+""" Renaming interface assets and splitting into direvtories """
+
+from .interfacehardcoded import ingame_hud_background_index
+
+
+def interface_rename(filename):
+    """
+    Returns a human-usable name according to the original and hardcoded metadata.
+    """
+    try:
+        return "interface/hud" + str(ingame_hud_background_index(filename)).zfill(4) + ".slp"
+    except ValueError:
+        return filename
```

Comment:
ok, but that's rather hacky :stuck_out_tongue_winking_eye: 

how about something like?

``` python
if filename in ingame_hud_background_index:
    return "interface/hud/%s.slp" % filename.zfill(4)
else:
    return filename
```